pyclam/utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+import numpy as np
+from scipy.special import erf
+
+SUBSAMPLE_LIMIT = 100
+BATCH_SIZE = 10_000
```

Comment:
I don't see a direct use of these in this file. Perhaps we should consider a `constants.py` file

Reply:
We can talk about a constants.py or a config.py file but I think that such a thing should be its own PR